os_brick/initiator/connectors/lightos.py

```diff
@@ -165,6 +161,11 @@ def monitor_message_queue(self, message_queue, lightos_db):
                 lightos_db[connection['uuid']] = connection
 
     def lightos_monitor(self, lightos_db, message_queue):
+        '''Bookkeeping lightos connections.
+
+        This is useful when the connector is comming up to a running node with
```

Comment:
This is useful when the connector comes up on a node that is already running with instances that may be connected to a working LightOS cluster. This method is used by the nova LightOS bits at \<pointer to the nova LightOS code\>.